static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss

```diff
@@ -148,9 +148,9 @@ $border-color: $fill-colour
 }
 .button--secondary {
     @include button-colour(
-        guss-colour(neutral-7, $pasteup-palette),
+        colour(button-mute),
```

Comment:
lol sorry missed this.

1. You can't reference frontend vars inside a component, otherwise it won't work outside of frontend.

2. Also you can't commit changes to the component from here. Get over to https://github.com/guardian/pasteup-buttons

soz again